src/assembler/rv_c_ext.h

```diff
@@ -445,7 +245,23 @@ class RVCInstrCAType : public RVCInstruction<Reg_T> {
                              RVCOpPartFunct2(funct2, RVCOpPartFunct2::OFFSET5),
                              RVCOpPartFunct6(funct6)),
             {std::make_shared<RVCRegRs2Prime<Reg_T>>(isa, 2),
-             std::make_shared<RVCRegRdRs1Prime<Reg_T>>(isa, 1)}) {}
+             std::make_shared<RVCRegRdRs1Prime>(isa, 1)}) {}
+
+  /// All RV-C Funct6 opcode parts are defined as a 6-bit field in bits 10-15
+  /// of the instruction
+  class RVCOpPartFunct6 : public RVCOpPart {
```

Comment:
Ubernit - when `class RVCOpPartFunct6` is nested within `class RVCInstrCAType`, prefixing the former with `RVC` feels redundant to me; this goes for all of the nested classes in this PR.